contracts/src/messaging/messenger/CrossChainEnabledObscuro.sol

```diff
@@ -71,4 +71,14 @@ abstract contract CrossChainEnabledObscuro is Initializable {
         );
         messageBus.publishMessage{value: value}(nonce++, topic, payload, consistencyLevel);
     }
+
+    // This function is used to publish a raw message instead of a function call to be relayed.
+    // This is useful for sending arbitrary data across chains.
+    function publishRawMessage(
```

Comment:
for consistency we should have: ``queueValueTransferCall`` and ``queueCrossChainCall``(above). 
The current `queueMessage` and `publishRawMessage` are not very explicit

Reply:
Only the bridge can do valueTransferCall so naming it like this makes no sense as users will be confused. What the function really do is publish a raw message without a wrapper.